src/main/java/org/aerogear/digger/client/model/BuildTriggerStatus.java

```diff
@@ -47,15 +49,19 @@
         /**
          * The build is stuck on Jenkins queue.
          */
-        STUCK_IN_QUEUE
+        STUCK_IN_QUEUE,
+
```

Comment:
Maybe add a test description here of when this enum is set

Reply:
updated